web/cypress/support/index.js

```diff
@@ -17,6 +17,14 @@
 /* eslint-disable no-undef */
 import './commands';
 import '@cypress/code-coverage/support';
+const { SERVE_ENV = 'dev' } = Cypress.env();
+
+before(() => {
+  // reset etcd before test
+  if (SERVE_ENV === 'test') {
+    cy.exec('etcdctl del --prefix /', { failOnNonZeroExit: false });
```

Comment:
it seems that the prefix for apisix data stored in etcd is `/apisix`, so is it necessary to specify the prefix to `/apisix`insread of `/`?
cc @nic-chen

Reply:
I think `/` is OK here for testing

Reply:
got it